compiler/luci/partition/src/ConnectNode.test.h

```diff
@@ -47,6 +47,38 @@ template <unsigned N> class TestIsOGraph : public TestIsGraphlet<N>, public Test
   }
 };
 
+template <class T> class NodeGraphletT
+{
+public:
+  virtual void init(loco::Graph *g)
+  {
+    _node = g->nodes()->create<T>();
+    _node->dtype(loco::DataType::S32);
+    _node->name("node");
+  }
+
+  T *node(void) const { return _node; }
+
+protected:
+  T *_node{nullptr};
+};
```

Comment:
I think this could be substituted by `NodeIsGraphletT<1>`.

Reply:
Yes, you are right. But for reading the code, without `<1>` would be better...